### PR TITLE
Documentation Updates

### PR DESCRIPTION
**Pull request by Matt-Gleich**
- PR template
- Refactored `CONTRIBUTING.md` for workshops and general contributions.
- Updated README.md accordingly

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,120 +1,54 @@
 # Contribution Guidelines
 
-This document contains a set of guidelines which any contributions to _any_ Hack Club repository must follow.
+## General Guidelines
 
-[If you disagree with something here, you should open up an issue to discuss making a change!](https://github.com/hackclub/hackclub/issues/new)
+- Conversations in pull requests and issues must align with the [hack club code of conduct](https://hackclub.com/conduct/).
```
Comment (itsmingjie) [inline CONTRIBUTING.md:5]: "Hack Club" should be capitalized =D

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,120 +1,54 @@
...
+### 🛠 Writing the Workshop
 
-File names should follow the following guidelines:
+Now we can get to making! Here are some workshop requirements:
 
-- All lowercase
-- `_` as space separator for file names
-- `-` as a space separator for dates in file names
-- Dates in `YY-MM-DD` format
+- Complies with the [hack club code of conduct](https://hackclub.com/conduct/).
```
Comment (lachlanjc) [inline CONTRIBUTING.md:30]: Capitalization here too

**Review by lachlanjc — COMMENTED** [review]
Awesome start here. Can we link to the workshop bounty guidelines directly though? I'm making a bunch of updates to those & don't want to fragment them across multiple documents except the PR template if needed.

**Comment by Matt-Gleich** [comment]
@lachlanjc these are the guidelines, right? https://workshops.hackclub.com/workshop-bounty/

**Comment by Matt-Gleich** [comment]
@lachlanjc any update on this?

File: README.md [hunk 3]
```diff
@@ -22,24 +22,9 @@ Hack Club was founded by [a teenager](https://zachlatta.com) who dropped out of
...
 | Follow us on Twitter            | <https://twitter.com/hackclub>       |
 
-## Contributing to our [Workshops](https://workshops.hackclub.com/)
+## Contributing
 
-Contributions are welcome!
-
-If you need any help, please contact us at team@hackclub.com or on our [Slack](https://slack.hackclub.com/).
-
-1. Check out our [public issues board][0]. If your issue isn't on the board, [open a new one][1].
```
Comment (lachlanjc) [inline README.md:31]: Can we keep some of these directions that are still relevant? (Not this first item.) Personally I'm a lot less likely to click through & read another file

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,120 +1,36 @@
...
-> Add the spark reactor source code into the document where we add our thoughts
-
-- This commit message is too long! It should be less than 50 characters!
-- It uses too many words and could be simplified
-
-## Making a Pull Request
-
-Here are a series of GIFs that illustrate how to make and commit a change, and create a pull request for review, using GitHub's web interface.
-
-![](img/gh_web_find_file.gif)
```
Comment (lachlanjc) [inline CONTRIBUTING.md:114]: Can you delete these image files if we're not using them?

**Review by lachlanjc — COMMENTED** [review]
Appreciated! A few notes/suggestions—

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,120 +1,36 @@
...
-1. [GitHub Flow](#github-flow)
-2. [Branch Names](#branch-names)
-3. [File Names](#file-names)
-4. [Git Commits](#git-commits)
-5. [Making a Pull Request](#making-a-pull-request)
+Hey there! Super excited to see that you want to contribute to a workshop. Before we get started make sure you have the following:
 
-## GitHub Flow
+- A GitHub account - [Sign Up](https://github.com/join)
+- Basic markdown knowledge - [Learn Markdown](https://www.markdowntutorial.com/)
```
[the remark this answers is not in the document]
Comment (Matt-Gleich) [reply CONTRIBUTING.md:14]: @lachlanjc are you looking at the latest changes on my doc-updates branch? This isn't in my changes

**Comment by faisalsayed10** [comment]
Hey, I see that this PR is now stale for a long time. Just to clear the backlog, I'm closing this for now, feel free to reopen! :)

**Pull request closed without merging**